io/plugins/eu.esdihumboldt.hale.io.jdbc.ui/src/eu/esdihumboldt/hale/io/jdbc/ui/SchemasRetrievalPage.java

```diff
@@ -106,71 +118,146 @@ public boolean updateConfiguration(ImportProvider provider) {
 	@Override
 	protected void createContent(Composite page) {
 		page.setLayout(GridLayoutFactory.swtDefaults().numColumns(1).create());
-		schemaList = new ListViewer(page, SWT.MULTI | SWT.V_SCROLL | SWT.H_SCROLL);
-//		schemaList.setFilters(filters);
-		schemaList.setLabelProvider(new LabelProvider() {
-
-			@Override
-			public String getText(Object element) {
-				return element.toString();
-			}
-
-		});
-		schemaList.setContentProvider(ArrayContentProvider.getInstance());
-		GridData layoutData = new GridData(SWT.FILL, SWT.FILL, true, false, 2, 1);
-		layoutData.widthHint = SWT.DEFAULT;
-		layoutData.heightHint = 8 * schemaList.getList().getItemHeight();
-		schemaList.getControl().setLayoutData(layoutData);
-		setPageComplete(false);
+		this.page = page;
 	}
 
 	/**
 	 * @see eu.esdihumboldt.hale.ui.HaleWizardPage#onShowPage(boolean)
 	 */
 	@Override
 	protected void onShowPage(boolean firstShow) {
-		if (firstShow) {
-			setPageComplete(true);
-		}
 		try {
+			// load configuration
+			loadConfiguration();
+
+			// if page is enable for driver
+			if (!isEnable) {
+				disposeControl();
+				// message
+				Label label = new Label(page, SWT.WRAP);
+				label.setText("Schema retrieval is not enabled for selected driver.");
```

Comment:
What the user is interested here is selecting schemas not retrieving schemas - I think this message would be rather confusing.
Maybe a message like "Restricting the loaded tables to specific database schemas is not enabled for this driver. All database tables will be loaded."